client-v2/src/util/pollingConfig.ts

```diff
@@ -1,14 +1,23 @@
 import { fetchStaleOpenCollections } from 'actions/Collections';
 import { Dispatch } from 'types/Store';
 import { Store } from 'types/Store';
+import { frontsEditPath } from 'components/App';
+import { matchPath } from 'react-router';
 
 /**
  * TODO: do we want to check if there are any collectionUpdates going out here
  * could there be a race condition where we try to update a collection and poll
  * at the same time and the poll overwrites the update
  */
 export default (store: Store) =>
-  setInterval(
-    () => (store.dispatch as Dispatch)(fetchStaleOpenCollections()),
-    10000
-  );
+  setInterval(() => {
```

Comment:
Previously, collection polling was asking for collections from the open fronts across all priorities. By deriving the current priority from the path, we can just poll for the collections open for the current priority.